src/components/Nav.svelte

```diff
@@ -14,7 +14,10 @@
   let windowWidth: number | undefined;
 </script>
 
-<svelte:window bind:innerWidth={windowWidth} />
+<svelte:window
+  bind:innerWidth={windowWidth}
+  on:sveltekit:navigation-end={() => (showMobileMenu = false)}
+/>
```

Comment:
Great work!

We can delete all the code in the comments above these lines now and thats within scope of this PR.

```js
  // let oldSegment: string;

  let showMobileMenu = false;

  // $: {
  //   if (segment !== oldSegment || (path && !path.startsWith(segment))) {
  //     showMobileMenu = false;
  //   }
  // }
```